tcl/Util.tcl

```diff
@@ -207,6 +210,29 @@ proc sta_warn_error { msg_id warn_error msg } {
   }
 }
 
+define_cmd_args "suppress_msg" {msg_ids}
+
+proc suppress_msg { args } {
+  variable suppressed_msgs
+  check_argc_eq1 "suppress_msg" $args
```

Comment:
Instead of taking a list of ids as one arg I think it should just use the args.
So
suppress_msg 1 2 3
instead of 
suppress_msg {1 2 3}